src/beta_inc.jl

```diff
@@ -0,0 +1,876 @@
+using Base.Math: @horner
+using Base.MPFR: ROUNDING_MODE
+const exparg_n = log(nextfloat(floatmin(Float64)))
+const exparg_p =  log(prevfloat(floatmax(Float64)))
+
+#COMPUTE log(gamma(b)/gamma(a+b)) when b >= 8
+"""
+    loggammadiv(a,b)
+
+Computes ``log(\\Gamma(b)/\\Gamma(a+b))`` when b >= 8
+"""
+function loggammadiv(a::Float64, b::Float64)
+    if a > b
+        h = b/a
+        c = 1.0/(1.0 + h)
+        x = h/(1.0 + h)
+        d = a + (b - 0.5)
+        x² = x*x
```

Comment:
this line and below are shared with the `else` branch, so can be just moved below.